docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
 Tasks and Flows
 ---------------
 
-CumulusCI uses a framework of *tasks* and *flows* to organize the automation that is available to each project.
+CumulusCI uses a framework of **tasks** and **flows** to organize the automation that is available to each project.
+
+Tasks are units of automation. A task could perform a deployment, load a dataset, retrieve material from an org, install a managed package, or undertake many other activities. CumulusCI ships with scores of tasks out of the box.
+
+Popular tasks:
+
+* ``cci task list``: Review the tasks available in a project.
+* ``cci task info <name>``: Learn more about a task and how to configure its options; ``<name>`` is the name of the task.
+* ``cci task run <name> --org <org>``: Run a task; ``<name>`` is the name of the task and ``<org>`` is the org you'd like to run it against.
 
-Tasks are units of automation. A task could perform a deployment, load a dataset, retrieve material from an org, install a managed package, or undertake many other activities. CumulusCI ships with scores of tasks out of the box. 
+        Example: ``run_tests`` executes Apex unit tests. If you have an org called ``dev``, you can run this task against it with the command ``cci task run run_tests --org dev``
 
-You can review the tasks available in a project by running ``cci task list``, learn more about a task and how to configure its options with ``cci task info <name>``, where ``<name>`` is the name of the task, and run a task with ``cci task run <name> --org <org>``, where ``<name>`` is the name of the task and ``<org>`` is the org you'd like to run it against. For example, the ``run_tests`` task executes Apex unit tests. If you have an org called ``dev``, you can run this task against this org with the command ``cci task run run_tests --org dev``.
+Many operations that you'll undertake with CumulusCI, including creating new orgs, use flows. Flows are ordered sequences of tasks (and other flows!) that produce a cohesive outcome, such as an org that's configured to suit a specific workflow.
 
-Many operations that you'll undertake with CumulusCI, including creating new orgs, use flows. Flows are ordered sequences of tasks (and other flows!) that produce a cohesive outcome, such as an org that's configured to suit a specific workflow. 
+Popular flows:
 
-Find the list of flows available in a project by running ``cci flow list``. Learn more about a flow and the tasks it contains with ``cci flow info <name>``, where ``<name>`` is the name of the flow, and run a flow with ``cci flow run <name> --org <org>``, where ``<name>`` is the name of the flow and ``<org>`` is the org you'd like to run it against.
+* ``cci flow list``: Review the flows available in a project.
+* ``cci flow info <name>``: Learn more about a flow and the tasks it contains; ``<name>`` is the name of the flow
+* ``cci flow run <name> --org <org>``: Run a flow; ``<name>`` is the name of the flow and ``<org>`` is the org you'd like to run it against.
```

Comment:
I am severely resistant to having it all lowercase. This is technically its own sentence, not an extension of one. 

Does it help to note that the reason why I put in colons was to save writing "allows user to" each time, which looks a bit goofy when repeated in a list. We can default to "reviews/learns/runs", but "learns more about a flow" or "runs a flow" doesn't hit my ear right when read aloud.

@lianeyee, thoughts?

Reply:
How about 

```suggestion
* ``cci flow run <name> --org <org>``: Run the flow ``<name>`` against the org ``<org>``.
```

Reply:
It's more succinct, but now it might not also be as clear to the reader. (This is where we start getting into the debate of who are we writing this for, and how much do they know or can intuit.)

Can we get more of the team to chime in on this one? @davisagli? @Br4nd0R?

Reply:
@justinvholt I agree with using the colon to economize on words. (Another option we sometimes use for such lists is an em dash, but I wouldn't here because a dash seems slightly less clear where the command stops and the separator starts.) 

For me, it is clearer to see the info spelled out in two separate parts. "Here's what the flow does. Here are the variables you specify." How about replacing the semicolons with periods? Seems simpler. And use ending periods on all. 

For extra succinctness, omit "you'd like."

Reply:
I like the more suggestion by @davidmreed. True, it may not be as clear, but including an example (as is present just after the 'popular tasks' list) should clear that up.

Reply:
@Br4nd0R: What example would you like to insert?